modules/300-prometheus/crds/clusteralerts.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusteralerts.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: prometheus
+    app: alerts-receiver
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: clusteralerts
+    singular: clusteralert
+    kind: ClusterAlert
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+      - name: Alert
+        type: string
+        jsonPath: .alert.name
+      - name: Severity
+        type: string
+        jsonPath: .alert.severityLevel
+      - name: Age
+        type: date
+        jsonPath: .status.startsAt
+      - name: Last received
+        type: date
+        jsonPath: .status.lastUpdateTime
+      - name: Status
+        type: string
+        jsonPath: .status.alertStatus
+      subresources:
+        status: {}
+      schema:
+        openAPIV3Schema:
+          type: object
+          description: |
+            Resource to display Prometheus alerts
+          required:
+            - alert
+          properties:
+            metadata:
+              type: object
+              properties:
+                name:
+                  type: string
+                  pattern: "^[a-z0-9]+$"
+                  maxLength: 64
```

Comment:
Add `description` with a comment about AlertManager fingerprint

Reply:
IMHO, the value of the name is more like an identifier than a name.

Reply:
added